src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,38 @@
+.watch {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  @include watch-parameters;
+  @include watch-background();
+
+  &__minute,
+  &__second,
+  &__center,
+  &__hour {
+    display: flex;
+    position: absolute;
+  }
+
+  &__minute,
+  &__second,
+  &__hour {
+    box-shadow: -2px 2px 1px 0 rgba(0,0,0,0.3);
+  }
+
+  &__hour {
+    @include hour;
+  }
+
+  &__minute {
+    @include minute;
+  }
+
+  &__second {
+    @include second;
+  }
+
+  &__center {
+    @include watch-center;
+  }
```

Comment:
I'm not sure it's good idea to use mixins here
check screen below, sass documentation describes that we should use mixins for reusable styles, but this is not our case. 
<img width="974" alt="Screenshot 2022-12-05 at 10 57 19 AM" src="https://user-images.githubusercontent.com/60295799/205595617-d66a59bf-505d-427e-a702-b942077464f6.png">